### ISSUE TITLE
child_process.exec output code is incomplete

### ISSUE THREAD

**Issue by TOP-Chao** [sass/node-sass#1793]
**node.js**：v7.0.0
**node-sass**: 3.11.2

```javascript
exec(cmd, {
    maxBuffer: 5000 * 1024,
    env: sass.env
}, (err, stdout, stderr)=> {
    console.log(stdout);
});
```
**command**：`/usr/local/lib/node_modules/node-sass/bin/node-sass --output-style compact /Users/cstyles/Project/heikuai/app-news/scss/styles.scss`
The console output is the complete code, but child_process.exec output code is incomplete

**Comment by xzyfer** [comment]
I don't understand what you're asking

**Comment by xzyfer** [comment]
This should be fixed by ##1823 which was released in v4.1.0.
